src/smokeTest/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsControllerSmokeTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.jobscheduler.controllers;
+
+import io.restassured.RestAssured;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+import org.springframework.http.HttpHeaders;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+public class JobsControllerSmokeTest {
+
+    @Before
+    public void setup() throws IOException {
+        Map<String, String> env = System.getenv();
+        if (env.get("TEST_URL") == null) {
+            Resource resource = new ClassPathResource("/smoke-application.properties");
+            Properties props = PropertiesLoaderUtils.loadProperties(resource);
+            RestAssured.baseURI = props.getProperty("default-test-url");
+        } else {
+            RestAssured.baseURI = env.get("TEST_URL");
+        }
+    }
```

Comment:
don't remember if i tried before, but does env var work with properties file? would be so much better just setting `base-url=${TEST_URL:http://localhost:8484}` and in here i believe you can pick config with `@Value()` magic provided by spring

Reply:
i need to boot up spring for that and all the crappy missing beans. not worth it just to basically do a curl request. @nitinprabhu and I tried for a while but no go